sentry-android-ndk/build.gradle.kts

```diff
@@ -38,6 +38,18 @@ android {
             cmake {
                 arguments.add(0, "-DANDROID_STL=c++_static")
                 arguments.add(0, "-DSENTRY_NATIVE_SRC=$sentryNativeSrc")
+
+                // https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md#additional-required-arguments
+                arguments.add(0, "-DANDROID_PIE=ON")
```

Comment:
Shouldn't be necessary given the `minSdkVersion` of 18 used for native code. The CMake toolchain should configure this for you.

Reply:
thanks :)